liquibase-core/src/main/java/liquibase/util/CommonsFilenameUtils.java

```diff
@@ -0,0 +1,902 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
We talked about only copying the code for the methods that we needed from FilenameUtils.  Is this all of them or the trimmed down version?

Reply:
Believe it or not this is the trimmed down version.